apps/condo/bin/lib/helpers.js

```diff
@@ -82,14 +86,34 @@ const getStartDates = (dateRaw) => {
     const date = getMonthStart(dateRaw)
 
     return {
+        prevMonthStart: date.subtract('1', 'month').format(DATE_FORMAT),
         thisMonthStart: date.format(DATE_FORMAT),
         nextMonthStart: date.add('1', 'month').format(DATE_FORMAT),
     }
 }
 
+/**
+ *
+ * @param context
+ * @returns {function(*, *, *, *=, *=): *[]}
+ */
+const getLoadByChunks = async (context) =>
```

Comment:
this duntion is redundant

Reply:
it allows to make more shorthand calls

for example 
```
const receipts = await loadByChunks(BillingReceipt, billingReceiptsWhere, ['createdAt_ASC'])
```
instead of
```
const receipts = await loadListByChunks({
    context: keystone,
    list: BillingReceipt, 
    where: billingReceiptsWhere, 
    sortBy: ['createdAt_ASC'])
}
```